documentation/installation/manual.md

```diff
@@ -30,30 +30,28 @@ With this content (**NB!: This was the content for 0.20.0, it may be different o
 
 **Do you see the `.translations` directory in the screenshot above? you _really really_ need that one.**
 
-### Step 3 - Move along the hacs folder to HA
+### Step 3 - Move the hacs folder to HA
 
 The folder named `hacs` needs to be copied to your Home Assistant installation.
 
-Use your favorite tool to move files to Home Assistant.
+Specifically, the `hacs` folder needs to be placed under `<config_dir>/custom_components/`.
 
-If this is your first custom_component you would need to create a new folder (see [step 4](#step-4---create-custom_components-folder)).
+If this is your first custom_component, that folder may not exist already. Go ahead and create it. The `custom_components` folder should exist in the same place as your `configuration.yaml` file.
 
-The `hacs` folder needs to be placed under `<config_dir>/custom_components/`
+Use your favorite tool to move the `hacs` folder to Home Assistant.
 
-On Home Assistant (supervised/docker) the final location will be `/config/custom_components/hacs`
+On Home Assistant (supervised/docker) the final location will be `/config/custom_components/hacs`.
 
-With a venv installation the final location will be `/home/homeassistant/.homeassistant/custom_components/hacs`
+With a venv installation the final location will be `/home/homeassistant/.homeassistant/custom_components/hacs`.
 
-### Step 4 - Create custom_components folder
+### Step 4 - Verify Permissions
 
-Open the folder where you have your `configuration.yaml` file.
+The user account running Home Assistant needs full control to the `custom_components` folder. Easiest way to do this is to have the user account that runs Home Assistant to own the `custom_components` folder.
 
-If you **do not** see a `custom_components` folder in **the same** folder as `configuration.yaml`, you need to create it.
+```
+sudo chown -R homeassistant:homeassistant custom_components/

```

Comment:
This only applies to venv installations that should be noted there like this:

```txt
    :::note
    If you are running Home Assistant in a python venv you need to make sure the user running Home Assistant has full control over these directories and files.
    
    ```bash
    sudo chown -R <name-of-user>:<name-of-group> <config_dir>/custom_components
    ```
    :::
```

wouldn't it be better to change to the `homeassistant` user during step 3?

Reply:
It would, but Step 3 started getting a little long, so I thought a separate step would break things up more.

I like the note, only a small group of people will actually need to do that, so that info is there if they need it.